src/components/Dashboard/TodoList.vue

```diff
@@ -1576,14 +1588,6 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, N
 
     if (header) {
       let subtitle: string = null
```

Comment:
This is probably obsolete now. I think you can also delete line 1608.

Reply:
Secondly, I think line 1609 should be the authorization enum. (Draft title is used for delete confirmation. Please test this.)

Reply:
Removed